cut-new-version.sh

```diff
@@ -5,7 +5,7 @@ version=$2
 
 echo 'Cutting a new version...'
 
-docusaurus docs:version:$product $version
+npx docusaurus docs:version:$product $version
```

Comment:
Was this change intended? 

I'm not sure (I'm completely unfamiliar with the new docs tooling), but just wanted to check.